server/plugin/command.go

```diff
@@ -342,22 +397,80 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		msg += "\n\n**Warning:** You subscribed to a private repository. Anyone with access to this channel will be able to read the events getting posted here."
 	}
 
-	return msg
+	if appErr = p.CreatePost(args.ChannelId, p.BotUserID, msg); appErr != nil {
+		return fmt.Sprintf("%s error creating the public post: %s", msg, appErr.Error())
+	}
+
+	return ""
 }
 
+func (p *Plugin) getSubscribedFeatures(channelID, owner, repo string) (Features, error) {
+	var previousFeatures Features
+	subs, err := p.GetSubscriptionsByChannel(channelID)
+	if err != nil {
+		return previousFeatures, err
+	}
+
+	for _, sub := range subs {
+		label := repo
+		if owner != "" {
+			label = owner + "/" + repo
+		}
+
+		if sub.Repository == label {
+			previousFeatures = sub.Features
+			return previousFeatures, nil
+		}
+	}
```

Comment:
The var name `label` can be confused with the GH concept of "label"

```suggestion
	for _, sub := range subs {
		fullRepoName := repo
		if owner != "" {
			fullRepoName = owner + "/" + repo
		}

		if sub.Repository == fullRepoName {
			return sub.Features, nil
		}
	}
```